templates/server-network-policy.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.openshift }}
```

Comment:
Same as above, seems like something for vanilla k8s as well.

Reply:
Same as previous comment. I didn't want to impact with this PR the way how Vault is deploy to vanilla K8S.